docs/css/extra.css

```diff
@@ -1,4 +1,32 @@
-@import "https://fonts.googleapis.com/icon?family=Material+Icons";
+@import 'https://fonts.googleapis.com/icon?family=Material+Icons';
+
+@font-face {
+    font-family: 'StJude Sans';
+    src: url('/_static/fonts/sjs-light.woff');
+    font-weight: 300;
+}
+
+@font-face {
+    font-family: 'StJude Sans';
+    src: url('/_static/fonts/sjs-light.woff');
+    font-weight: 300;
+}
+
+@font-face {
+    font-family: 'StJude Sans';
+    src: url('/_static/fonts/sjs-regular.woff');
+    font-weight: 400;
+}
+
+@font-face {
+    font-family: 'StJude Sans';
+    src: url('/_static/fonts/sjs-bold.woff');
+    font-weight: 700;
+}
+
+* {
+    font-family: "StJude Sans"
```

Comment:
`font-family` is an inherited property. This should be on an upper-level parent, e.g., `body`.